testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/UsedVersion.java

```diff
@@ -149,14 +149,16 @@ static MVersion parse(List<String> lines) {
                 String mandrelVersion = mandrelVersion(vendorVersion);
                 String versNum = (isMandrel(vendorVersion) ? mandrelVersion : graalVersion);
                 Version vers = versionParse(versNum);
+                final String lastLine = lines.get(lines.size() - 1).trim();
                 VersionBuilder builder = new VersionBuilder();
-                return builder.jdkUsesSysLibs(false /* not implemented */)
-                       .beta("" /* not implemented */)
-                       .jdkFeature(v.feature())
-                       .jdkInterim(v.interim())
-                       .jdkUpdate(v.update())
-                       .version(vers)
-                       .build();
+                return builder
+                        .jdkUsesSysLibs(lastLine.contains("-LTS"))
```

Comment:
I don't think this will be necessarily true. `LTS` in the version output `!=` whether or not system libs are being used. It seems safe to assume that lastest releases no longer use system libs. Not sure if this condition still has any meaningful value.

Reply:
It is used to distinguish Mandrel based on Red Hat build of OpenJDK that has this different sys libs behavior from other distros.

Reply:
@jerboaa i.e. with "LTS" in there, we skip doing hashsum checks due to slightly different anti aliasing on fonts...
https://github.com/Karm/mandrel-integration-tests/blob/master/testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java#L583

While I am writing this, I wonder if it's something to be revisited again...  (not in this PR though I guess)